### ISSUE TITLE
Legacy API removed from Danbooru; no longer working

### ISSUE THREAD

**Issue by JMM72** [Nandaka/DanbooruDownloader#108]
A very recent update in Danbooru removed the Legacy API which DanbooruDownloader is using right now. This makes DD unable to download from Danbooru (sad irony). Other providers work, tested with Sankaku Complex, Konachan, Gelbooru and Yande.re for a start (since these are still using the Legacy API).

For now it's not sure if they will bring it back again (one serious request for IQDB.org). In any case, it's mentioned it has been deprecated for years and everyone should be using the current API.

[Forum post about the issue, look at the bottom of the page.](http://danbooru.donmai.us/forum_topics/9127?page=153)

**Comment by Nandaka** [comment]
Looks like they enable the old api again.

**Comment by JMM72** [comment]
Yes but it's a temporal measure. They want to remove access to the legacy API due to resource hog whenever it's used, so updating to the new API should be a priority.

**Comment by Nandaka** [comment]
Related to #128
